packages/react/src/MeasureReportDisplay/MeasureReportDisplay.tsx

```diff
@@ -0,0 +1,39 @@
+import { Box, SimpleGrid } from '@mantine/core';
+import { Measure, MeasureReport, MeasureReportGroup, Reference } from '@medplum/fhirtypes';
+import { useMedplum, useResource } from '@medplum/react-hooks';
+import React, { useEffect } from 'react';
+import { MeasureReportDisplayGroup, MeasureTitle } from './MeasureReportGroup/MeasureReportGroup';
+
+export interface MeasureReportDisplayProps {
+  measureReport: MeasureReport | Reference<MeasureReport>;
+}
+
+export function MeasureReportDisplay(props: MeasureReportDisplayProps): JSX.Element | null {
+  const medplum = useMedplum();
+  const report = useResource(props.measureReport);
+  const [measure, setMeasure] = React.useState<Measure | undefined>();
```

Comment:
You might need to test it, but I'm pretty sure you can use the new `useSearchOne` hook here.

Something like:

```tsx
const [measure] = useSearchOne('Measure',  { url: report?.measure });
```

Then you wouldn't need the extra `useState` and `useEffect`